source/math/FloatBox2D.ooc

```diff
@@ -58,14 +58,14 @@ FloatBox2D: cover {
 	}
 	pad: func ~fromFloat (pad: Float) -> This { this pad(pad, pad, pad, pad) }
 	pad: func ~fromSize (pad: FloatSize2D) -> This { this pad(pad width, pad width, pad height, pad height) }
-	pad: func ~fraction (pad: Float) -> This {
-		this pad(pad * this size / 2.0f)
+	enlargeEvenly: func (fraction: Float) -> This {
+		this pad(fraction * (this size width + this size height) / 2.0f)
 	}
-	padFractionAverage: func (pad: Float) -> This {
-		this pad(pad * (this size width + this size height) / 2.0f)
+	enlarge: func (fraction: Float) -> This {
+		this scale(1.0f + fraction)
 	}
-	shrink: func ~fraction (margin: Float) -> This {
-		this pad(-margin * this height / 2.0f)
+	shrink: func (fraction: Float) -> This {
+		this scale(1.0f - fraction)
```

Comment:
Maybe it would make sense to rename `enlarge` and `shrink` to `scaleUp` and `scaleDown` respectively since they are basically wrappers for `scale`?